ny-portal/src/app/(frontend)/layout.tsx

```diff
@@ -1,10 +1,8 @@
-"use client";
-
-import { CookieConsentProvider } from "@fremtind/jokul/components/cookie-consent";
 import Link from "next/link";
-import { Navigation } from "@/components/Navigation";
+import { Navigation } from "@/components/Navigation/Navigation";
```

Comment:
Kan jeg slå et slag for at vi bruker lowercase for directory navn under `@/components` og importerer via en indeks-fil?  Da blir det samme stil som vi har brukt ellers i repoet.

dvs

```suggestion
-import { Navigation } from "@/components/Navigation/Navigation";
+import { Navigation } from "@/components/navigation";
```

Det er mer DRY pluss at en indeks-fil gjør det tydelig hvilke deler av komponenten det er ment at man skal bruke.